### ISSUE TITLE
Indonesian translation for Lesson Plan "Intro to the Site Editor and Templates "

### ISSUE THREAD

**Issue by kharissulistiyo** [WordPress/Learn#1488]
# Details
- Link to original content: https://learn.wordpress.org/lesson-plan/intro-to-site-editor-and-templates/
- Link to original content GitHub issue: 
- Language you'll be translating to: Indonesian
- Have you arranged for someone to review this translation?: Yes
- Reviewer's GitHub username: kharissulistiyo
- Other info: 

# Next Steps
Once translated, please link or upload your translated files in a comment on this issue, and request a [translation review](https://make.wordpress.org/training/handbook/content-localization/#translation-review).

**Comment by kharissulistiyo** [comment]
* Translation doc: https://docs.google.com/document/d/1jgXrCc-TriIEyVbK6PVJHnHV-XQI6mTFlgSDIzxN7hw/edit?usp=sharing

**Comment by kaitohm** [comment]
@ardianimaya Are you able to review this translation for the team?

**Comment by ardianimaya** [comment]
Hi Ben! This one's mostly translated by me. Is it allowed to review my own translation? @bsanevans

**Comment by kaitohm** [comment]
Oh, I see. In that case, we'd really like someone else to review it first. (I've added you as a assignee on this issue 😉 ) Are you able to find someone in the Indonesian Slack, @ardianimaya ?

**Comment by ardianimaya** [comment]
@bsanevans I've shared this in the Indonesian Slack, but seems like everyone was occupied with the anniversary last week. Let me try again :D

**Comment by kaitohm** [comment]
Were you able to find a reviewer for this, @ardianimaya ?

**Comment by ardianimaya** [comment]
@bsanevans Review done by @kharissulistiyo , will publish it next week.

**Comment by kaitohm** [comment]
Great to hear! Let me know if you need assistance with that :)

**Comment by ardianimaya** [comment]
Published. https://learn.wordpress.org/lesson-plan/id-intro-to-site-editor-and-templates/
